compiler/passes/errorHandling.cpp

```diff
@@ -396,3 +422,148 @@ void lowerErrorHandling() {
     fn->accept(&visitor);
   }
 }
+
+namespace {
+
+class CanThrowVisitor : public AstVisitorTraverse {
+
+public:
+  CanThrowVisitor       (bool inThrowingFn, bool makeCompileErrors);
+
+  virtual bool enterTryStmt  (TryStmt*   node);
+  virtual void exitTryStmt   (TryStmt*   node);
+  virtual bool enterCatchStmt(CatchStmt*   node);
+  virtual void exitCatchStmt (CatchStmt*   node);
```

Comment:
You can remove the `enter` and `exitCatchStmt` declarations, since you're just using `AstVisitorTraverse` implementations.